### PR TITLE
Simplify test.

### PR DESCRIPTION
**Pull request by kdashg**

### CODE REVIEW

File: sdk/tests/conformance/rendering/point-size-varyings.html [hunk 1]
```diff
@@ -35,76 +33,68 @@
...
+varying mediump vec2 vPos;
+
 void main()
 {
-    gl_FragColor = vec4(vUV.xy,0.0,1.0);
-    if (isCloud == 1.0) {
-      gl_FragColor = vec4(gl_PointCoord.xy,0.0,1.0);
+    gl_FragColor = vec4(vPos * -2.0, 0, 1);
+
+    if (defaultsToZero == 1.0) {
```
Comment (zhenyao) [inline sdk/tests/conformance/rendering/point-size-varyings.html]: even this is not needed.  Simply put gl_FragColor = vec4(gl_PointCoord.xy, 0, 1) before gl_FragColor = vec4(vPos \* -2.0, 0, 1) is enough to trigger this bug.

**Comment by kdashg** [comment]
Indeed!

**Comment by zhenyao** [comment]
Looks good to me.

**Comment by emnh** [comment]
@jdashg Your PR doesn't check that the varying actually varies (being interpolated), does it? Shouldn't it? To do so it needs to check the color of an interior point of the triangle. I don't know if multiple interpolation schemes are allowed or if it is strictly specified, but then maybe it could check that the color is in between corners, instead of testing a fixed color. A possible bug could be that because gl_PointSize/gl_PointCoord is mentioned that only the corners are correctly colored.

**Comment by emnh** [comment]
@jdashg Ah sorry, it does. I didn't understand it correctly.

**Pull request merged**